sql/catalyst/src/main/antlr4/org/apache/spark/sql/catalyst/parser/SqlBase.g4

```diff
@@ -55,6 +55,7 @@ statement
         rowFormat?  createFileFormat? locationSpec?
         (TBLPROPERTIES tablePropertyList)?
         (AS? query)?                                                   #createTable
+    | createTableHeader LIKE source=tableIdentifier                    #createTableLike
```

Comment:
actually, what does it mean to provide table properties or row format in a `CREATE TABLE LIKE` statement? Trying to merge the provided properties with the existing ones seems complicated and full of corner cases. It might be simpler to just handle `CREATE TABLE x LIKE y` instead of accepting more things that we don't know how to handle.